utils/blech_post_process_utils.py

```diff
@@ -982,19 +993,20 @@ def ask_split(self):
                 check_func = lambda x: x in ['y','n'],
                 fail_response = 'Please enter (y/n)')
         if continue_bool:
-            if msg == 'y': 
+            if msg == 'y':
                 self.split = True
             elif msg == 'n': 
                 self.split = False
 
     def check_split_sort_file(self):
```

Comment:
I started working on the suggested edit here, and that proved really helpful; it turned out that not only were spaces problematic in the CSV, if the user intentionally supplied a non-numeric value to Split, like "y" or "x" for instance, it would initiate splitting and then throw an error. Fixed now.